application/forms.py

```diff
@@ -12,14 +12,14 @@
 import pytz
 
 from flask_wtf import FlaskForm
-from wtforms import validators, SelectField, TextField, IntegerField, DateTimeField, FloatField, BooleanField
+from wtforms import validators, SelectField, TextField, IntegerField, DateTimeField, BooleanField, DecimalField
```

Comment:
You can alphabetise these if it's bothering you. :)

Reply:
I'm about to do another PR for linting so I'll do it there 😛